infra/src/Pos/Infra/Communication/Types/Protocol.hs

```diff
@@ -186,8 +186,9 @@ buildBS :: ByteString -> B.Builder
 buildBS = bprint base16F
 
 data HandlerSpec
-    = ConvHandler { hsReplyType :: MessageCode }
-    | UnknownHandler Word8 ByteString
+    -- | ConvHandler hsReplyType
+    = ConvHandler !MessageCode
+    | UnknownHandler !Word8 !ByteString
     deriving (Show, Generic, Eq)
 
 instance Bi HandlerSpec where
```

Comment:
We don't seem to have golden/bitripping tests for `HandlerSpec`. At least not in the `infra` package.

We need that before we go changing things.

Reply:
Will do, I'll create a ticket.

Reply:
Tests created in https://github.com/input-output-hk/cardano-sl/pull/3268 and merged. Rebased [CDEC-403] onto develop with golden and bitripping tests still passing.